APIs_Drivers/UnbufferedSerial/main.cpp

```diff
@@ -0,0 +1,35 @@
+#include "mbed.h"
+
+
+// Create a DigitalOutput object to toggle an LED whenever data is received
+static DigitalOut led_serial_read(LED1);
+
+// Create a new UnbufferedSerial object with a default baud rate.
+static UnbufferedSerial serial_port(USBTX, USBRX);
+
+void on_rxd_interrupt()
```

Comment:
The most common abbreviation for receive is `rx`so I would name this function `on_rx_interrupt`